local.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:16.14.2-alpine
 
-ENV PORT 5040
+ENV PORT 5030
```

Comment:
Does 5040 clash an exising CRI?

Reply:
No but its what the core stub had for fraud. I couldnt see any clashes but if ive missed something let me know.

Reply:
re: ports - I'd normally say, let the other team change their values, but it doesn't matter too much here :)

You'll also need to update the `README.md`